internal/manifests/manifestutils/storage.go

```diff
@@ -107,7 +175,7 @@ func configureS3StorageWithCOOAuth(pod *corev1.PodSpec, containerIdx int, tempo
 
 	// Define volume with credentials
 	pod.Volumes = append(pod.Volumes, tokenCCOAuthConfigVolume(tempo))
-	pod.Volumes = append(pod.Volumes, saTokenVolume())
+	pod.Volumes = append(pod.Volumes, saTokenVolume(awsDefaultAudience))
```

Comment:
awsDefaultAudience ? is this correct?

Reply:
Yes, I have to add the audience attribute to the function, so here I'm just passing the awsdefaultAudience to the s3 configuration.